tests/integration_test.go

```diff
@@ -0,0 +1,159 @@
+// +build integration
+
+package tests
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+const (
+	diffBase     = "gcr.io/gcp-runtimes/diff-base"
+	diffModified = "gcr.io/gcp-runtimes/diff-modified"
+
+	aptBase     = "gcr.io/gcp-runtimes/apt-base"
+	aptModified = "gcr.io/gcp-runtimes/apt-modified"
+
+	// Why is this node-modified:2.0?
+	nodeBase     = "gcr.io/gcp-runtimes/node-modified:2.0"
```

Comment:
hmm...I wonder if the source code for these images is lying around anywhere, I'm not a huge fan of this being abstracted

Reply:
Yeah as a TODO:  get rid of these + get rid of the `_expected.json` files.